spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -349,9 +349,23 @@
     end
   end
 
-  def visit_idp_from_sp_with_loa1
+  context 'going back to the SP' do
+    it 'links back to the SP from the sign in page' do
+      state = SecureRandom.hex
+
+      visit_idp_from_sp_with_loa1(state: state)
+
+      click_link t('links.sign_in')
+
+      expect(page).to have_link(
+        t('links.back_to_sp', sp: 'Test SP'),
+        href: "http://localhost:7654/auth/result?error=access_denied&state=#{state}"
```

Comment:
this string feels a bit mysterious here....maybe if we named it that would be helpful?